Dockerfile

```diff
@@ -9,6 +9,15 @@ COPY internal/ internal/
 ARG KUBEBENCH_VERSION
 RUN make build && cp kube-bench /go/bin/kube-bench
 
+# Add kubectl to run policies checks
+ARG KUBECTL_VERSION TARGETARCH KUBECTL_SUM_amd64 KUBECTL_SUM_arm64 KUBECTL_SUM_s390x KUBECTL_SUM_ppc64le
+ADD --chown=root:root --chmod=0755 \
+    "https://dl.k8s.io/release/v${KUBECTL_VERSION}/bin/linux/${TARGETARCH}/kubectl" \
+    /usr/local/bin/kubectl
```

Comment:
How about download the checksum file and kubectl binary via wget or curl? 
By this way, we no longer need `KUBECTL_SUM_amd64`,  `KUBECTL_SUM_arm64`, `KUBECTL_SUM_s390x`   and `KUBECTL_SUM_ppc64le` arguments:

```
RUN wget -O /usr/local/bin/kubectl "https://dl.k8s.io/release/v${KUBECTL_VERSION}/bin/linux/${TARGETARCH}/kubectl"
RUN wget -O kubectl.sha256 "https://dl.k8s.io/release/v${KUBECTL_VERSION}/bin/linux/${TARGETARCH}/kubectl.sha256"
RUN sha256sum xxx
RUN chmod +x /usr/local/bin/kubectl
```

Reply:
> How about download the checksum file and kubectl binary via wget or curl? By this way, we no longer need `KUBECTL_SUM_amd64`, `KUBECTL_SUM_arm64`, `KUBECTL_SUM_s390x` and `KUBECTL_SUM_ppc64le` arguments:
> 
> ```
> RUN wget -O /usr/local/bin/kubectl "https://dl.k8s.io/release/v${KUBECTL_VERSION}/bin/linux/${TARGETARCH}/kubectl"
> RUN wget -O kubectl.sha256 "https://dl.k8s.io/release/v${KUBECTL_VERSION}/bin/linux/${TARGETARCH}/kubectl.sha256"
> RUN sha256sum xxx
> RUN chmod +x /usr/local/bin/kubectl
> ```

See all suggestions update in the last commit https://github.com/aquasecurity/kube-bench/commit/2e05ad3f1fe3ee128e1675c26b8c81c5d0d0d0db.
- Used `go env GOARCH`
- Removed the use of SUM per ARCH in Dockerfile, publish.yaml and Makefile
- Removed `TARGETARCH` in publish.yaml